txnsync/bitmask_test.go

```diff
@@ -145,7 +147,9 @@ func trimIterateHelper(t *testing.T, setBits []int) {
 
 	// For types 2, let the sum exceed entries
 	if int((b)[0]) == 2 {
-		require.Equal(t, errIndexNotFound, b.iterate(setBits[len(setBits)-1], len(setBits)-1, iterfunc))
```

Comment:
This should be an error according to how iterate is used in encodedgroupsunmarshalers.go